src/middlewares/error.middleware.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Request, Response } from 'express';
+import { Middleware, ExpressErrorMiddlewareInterface } from 'routing-controllers';
+import { TESTING_ENV } from '@config';
+import { Service } from 'typedi';
+
+interface ErrorInterface extends Error {
+  httpCode:number;
+}
+
+@Middleware({ type: 'after' })
+@Service()
+export class ErrorMiddleware implements ExpressErrorMiddlewareInterface {
+  // eslint-disable-next-line max-len
```

Comment:
is this disable necessary? I think prettier should format accordingly

Reply:
I didn't want to change the linter rules that were set, but yes, there's a max length per line of 90, and this line has 109
![image](https://user-images.githubusercontent.com/28785574/116120800-52d61880-a696-11eb-9822-2c61b057ef4e.png)

Reply:
Yes. I mean that the arguments should've gotten split into a couple of lines automatically

Reply:
ahhh gotcha, you are right, I was using eslint, not pretty, thanks!